docs/topics/mta-rn-known-issues-6-2-0.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-6-2-0_{context}"]
+= Known issues
+
+MTA version 6.2.0 has the following issues.
+
+.Analysis fails when fetching rules from a repository with a folder that contains spaces in its name
+
+When fetching custom rules from a repository during an analysis, if the `Root path` field contains spaces, the `mta-cli` command is not properly composed and the analysis fails. Spaces in directories specified in `userRulesDirectory` cannot be supported because it accepts a space-delimited list of directory paths. link:https://issues.redhat.com/browse/MTA-458[MTA-458]
+
+.Update notifications are disabled for Application, Job functions, and Business services
+
+Update notifications are disabled for **Application**, **Job function** and **Business services**, as a result, no notifications are displayed. link:https://issues.redhat.com/browse/MTA-1024[MTA-1024]
+
+.Repository type field is not required
+
+The **Repository type** field is not required when saving the configuring rules files from a repository in analysis. link:https://issues.redhat.com/browse/MTA-1047[MTA-1047]
+
+.False 'not connected' message for new Jira instance
```

Comment:
@anarnold97 could you please change it to: False 'not connected' status...?
Thanks